test/libyul/Parser.cpp

```diff
@@ -44,6 +44,9 @@ using namespace solidity;
 using namespace solidity::util;
 using namespace solidity::langutil;
 
+BOOST_TEST_DONT_PRINT_LOG_VALUE(ErrorId)
+BOOST_TEST_DONT_PRINT_LOG_VALUE(Error::Type)
```

Comment:
We don't have `<<` for these defined. I can either define it or tell boost not to try to print it. For now I did the latter.